src/mentors.json

```diff
@@ -506,8 +506,8 @@
     },
     {
       "name": "Felicia Ng",
-      "role": "Chief of Operations and External Relations",
-      "organization": "INTELLEX",
+      "role": "Chief, People Operations & External Relations",
+      "organization": "INTELLLEX",
       "school": "Nanyang Technological University",
       "courseOfStudy": "Business Administration",
       "fullBio": "Felicia Ng is Chief of Operations and External Relations at INTELLEX \u2014 a legal-tech start-up in the business of solving \"knowledge management\"-related problems in professional firms, with its product being a knowledge management system powered by an intelligent search engine that deploys proprietary machine learning and natural language processing algorithms. She graduated from the Nanyang Technological University with a bachelor's degree in business administration, after which she took up a job as a Customer Affairs Manager with Singapore Airlines\u00a0\u2014\u00a0this was not her first job, though, for her foray into the working world began as early as the tender age of 15. Since working in customer success then, Felicia has gone on to hold appointments in Business Partnerships for Singapore Airlines, and Public Service Leadership at the Public Service Division. In the course of her work, however, she has undertaken a whole\u2014wider\u2014range of portfolios, from crisis management to policy-making and people operations, spanning a diversity of industries, such as aviation, government and technology. To Felicia, she observes the common thread in her professional experiences to be bringing people with different perspectives together to solve challenging problems. This motivates her, as she relishes the opportunity to help others maximise their growth potential for greater impact, and learn from the expertise and experiences of others, such as her own.",
```

Comment:
Biography should be changed to "INTELLLEX" too (three 'L's).

Reply:
Thanks for catching this